packages/test-utils/dist/vue-test-utils.iife.js

```diff
@@ -9015,7 +9015,10 @@ var VueTestUtils = (function (exports, Vue, vueTemplateCompiler) {
    */
```

Comment:
We usually don't modify the dist files by hand. Can you revert those? I think we only need to change is-visible.js.

The dist files are generated and committed when we do a release. Thanks.

Reply:
Got it :+1: